src/FsAutoComplete/CodeFixes/AdjustConstant.fs

```diff
@@ -15,9 +15,9 @@ open Microsoft.FSharp.Core.LanguagePrimitives
 ///
 /// Note: As constant, because F# doesn't have `#define`
 [<Literal>]
-let private DEBUG = false
```

Comment:
I was building with a 9 preview SDK and the inlining/visibility changes that Kevin made flagged a bunch of failed inlining in this file.